packages/experiments-realm/blog-post.gts

```diff
@@ -588,36 +595,165 @@ export class BlogPost extends CardDef {
       return 'Scheduled';
     },
   });
+  @field blog = linksTo(BlogAppCard);
+
+  get formattedDatePublished() {
+    if (this.status === 'Published' && this.publishDate) {
+      return formatDatetime(this.publishDate, {
+        year: 'numeric',
+        month: 'short',
+        day: 'numeric',
+      });
+    }
+    return undefined;
+  }
+
+  get datePublishedIsoTimestamp() {
+    if (this.status === 'Published' && this.publishDate) {
+      return this.publishDate.toISOString();
+    }
+    return undefined;
+  }
+
   static embedded = EmbeddedTemplate;
   static fitted = FittedTemplate;
   static isolated = class Isolated extends Component<typeof this> {
     <template>
       <article>
         <header>
-          <h1><@fields.title /></h1>
-          <p class='description'><@fields.description /></p>
-          <@fields.authorBio class='byline' />
+          <@fields.blog class='blog' @displayContainer={{false}} />
+          {{#if @model.thumbnailURL}}
+            <figure>
+              <img
+                class='featured-image'
+                src={{@model.thumbnailURL}}
+                alt='remote work'
+              />
+              <figcaption>
+                Success in remote work is achievable with the right approach.
+              </figcaption>
+            </figure>
+          {{/if}}
+          {{#if @model.title}}
+            <h1><@fields.title /></h1>
+          {{/if}}
+          {{#if @model.description}}
+            <p class='description'>
+              <@fields.description />
+            </p>
+          {{/if}}
+          <ul class='info'>
+            {{#if @model.authorBio.title}}
+              <li class='byline'>
+                <@fields.authorBio
+                  class='author'
+                  @format='atom'
+                  @displayContainer={{false}}
+                />
+              </li>
+            {{/if}}
+            {{#if @model.datePublishedIsoTimestamp}}
+              <li class='pub-date'>
+                Published on
+                <time timestamp={{@model.datePublishedIsoTimestamp}}>
+                  {{this.formattedDatePublished}}
+                </time>
+              </li>
+            {{/if}}
+          </ul>
         </header>
         <@fields.body />
       </article>
       <style scoped>
+        @import url('https://fonts.googleapis.com/css2?family=Lora:ital,wght@0,400..700;1,400..700&family=Playfair+Display:ital,wght@0,400..900;1,400..900&display=swap');
```

Comment:
It’s fine of course but this being repeated across the different views does make me wish we could share things more easily.